.github/workflows/cargo-audit.yaml

```diff
@@ -0,0 +1,23 @@
+name: "Audit Dependencies"
+
+on:
+  push:
+    paths:
+      # Run if workflow changes
+      - '.github/workflows/cargo-audit.yaml'
+      # Run on changed dependencies
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  # Rerun periodically to pick up new advisories
+  schedule:
+    - cron: '0 0 * * *'
+  # Run manually
+  workflow_dispatch:
```

Comment:
Why not running on new PRs? This seems to be the best way to miss problematic dependencies when we introduce new deps to the repo?